bCNC/lib/bpath.py

```diff
@@ -1119,6 +1143,52 @@ def invert(self):
 		self.extend(new)
 	reverse = invert
 
+	#----------------------------------------------------------------------
+	# Split path into contours
+	# This not only SPLITs path to contours,
+	# it also takes unsorted segments and JOINs them to closed loops if possible
+	# duplicate of split2contours ? but is does not affect the path, it makes a deepcopy of it
+	# not used yet
```

Comment:
if only difference from split2contours is the deepcopy, why not just do deepcopy and then call split2contours? I would really like to prevent duplicate code in bCNC core, because maintenence is already hard enough.

Reply:
Yes I rewrote split2contours for me and wanted to save it somewhere, I can remove it now